### ISSUE TITLE
Needs 1.17+ update

### ISSUE THREAD

**Issue by kongming819** [KovuTheHusky/dynmap-structures#47]
Dynmap Structures does not load on a 1.17 server, most likely due to the new biomes system in 1.17. Sounds like they may further change things in 1.18, drastically overhauling worldgen and biomes.

```
[ERROR] .... Error occurred while enabling dynmap-structures v2.3.1 (Is it up to date?)

java.lang.NullPointerException: Cannot read the array length because "com.kovuthehusky.dynmap.structures.DynmapStructuresPlugin.BIOMES[org.bukkit.block.Biome.ordinal()]" is null

at com.kovuthehusky.dynmap.structures.DynmapStructuresPlugin.onEnable(DynmapStructuresPlugin.java:228) ~[?:?]

at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263) ~[patched_1.17.1.jar:git-Paper-127]

at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:370) ~[patched_1.17.1.jar:git-Paper-127]

at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:500) ~[patched_1.17.1.jar:git-Paper-127]

at org.bukkit.craftbukkit.v1_17_R1.CraftServer.enablePlugin(CraftServer.java:527) ~[patched_1.17.1.jar:git-Paper-127]

at org.bukkit.craftbukkit.v1_17_R1.CraftServer.enablePlugins(CraftServer.java:441) ~[patched_1.17.1.jar:git-Paper-127]

at net.minecraft.server.MinecraftServer.loadWorld(MinecraftServer.java:639) ~[patched_1.17.1.jar:git-Paper-127]

at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:306) ~[patched_1.17.1.jar:git-Paper-127]

at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1126) ~[patched_1.17.1.jar:git-Paper-127]

at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:316) ~[patched_1.17.1.jar:git-Paper-127] 
```

**Comment by KovuTheHusky** [comment]
Will get to this ASAP! I've been running 1.16.x personally and it slipped my mind that 1.17.x has big changes despite being split in two!

**Comment by KovuTheHusky** [comment]
Fixed by b6def49b86a55d425732f3482f080ccc8e4b9158. Try updating to release [2.4.0](https://github.com/KovuTheHusky/dynmap-structures/releases/tag/2.4.0). Thanks for reporting the issue!
